src/gtest/test_merkletree.cpp

```diff
@@ -1,9 +1,11 @@
 #include <gtest/gtest.h>
 
 #include "test/data/merkle_roots.json.h"
+#include "test/data/merkle_roots_new.json.h"
```

Comment:
Yeah, #908 gets rid of the old tree anyway so I'll get rid of the `_new` ones and just test this tree against itself.